src/main/java/com/faforever/api/deployment/ExeUploaderController.java

```diff
@@ -32,6 +36,11 @@ public ExeUploaderController(
     this.exeUploaderService = exeUploaderService;
   }
 
+  @RequestMapping(path = "/upload_form", method = RequestMethod.GET, produces = TEXT_HTML_VALUE)
```

Comment:
I would rename this to just /upload . GET gives the form, POST does the action.

Reply:
Np